src/chttpd/src/chttpd_db.erl

```diff
@@ -569,15 +569,23 @@ db_req(#httpd{method='POST',path_parts=[_,<<"_bulk_docs">>], user_ctx=Ctx}=Req,
             send_json(Req, 417, ErrorsJson)
         end;
     false ->
-        case fabric:update_docs(Db, Docs, [replicated_changes|Options]) of
-        {ok, Errors} ->
-            chttpd_stats:incr_writes(length(Docs)),
-            ErrorsJson = lists:map(fun update_doc_result_to_json/1, Errors),
-            send_json(Req, 201, ErrorsJson);
-        {accepted, Errors} ->
-            chttpd_stats:incr_writes(length(Docs)),
-            ErrorsJson = lists:map(fun update_doc_result_to_json/1, Errors),
-            send_json(Req, 202, ErrorsJson)
+        case lists:any(fun(JsonObj) ->
```

Comment:
The case format looks a bit odd, it's a bit nicer to have the `case Exp of` all in one line. So we might do the check first then have something like `case HasMissingRev of`.

However, since we're throwing a `bad_request` anyway there is probably no reason to use a case statement. Just `lists:foreach(fun(Doc) -> ` might work?